src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,30 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import main.java.core.basesyntax.Lottery;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
```

Comment:
You need create three balls using class Lottery and print information about them in console
let's create constant for number of balls
create Lottery object
use for loop 
and System.out.println(lottery.getRandomBall());